lib/multigrid.cpp

```diff
@@ -1832,12 +1807,9 @@ namespace quda
     // now reallocate the B vectors copy in e-vectors
     for (int i = 0; i < (int)param.B.size(); i++) {
       param.B[i] = new ColorSpinorField(bParam);
-      *param.B[i] = *B_evecs[i];
+      *param.B[i] = B_evecs[i]; // FIXME can std::move this
```

Comment:
fix now or later?

Reply:
Leave it for later.  The multigrid.cpp handling of vector sets needs to be rewritten completely, and much smarter things can be done here.  I intentionally tried to minimize touching multigrid.cpp to minimize conflicts with #1283.

Moreover, that specific code needs to made more robust (e.g., what happens if `n_conv` isn't at least equal to `B.size()`?).  

My plan is to do a specific multigrid-focused cleanup PR that will also add proper regression tests.